src/components/Compare/LocationTable.tsx

```diff
@@ -257,11 +257,11 @@ const LocationTable: React.FunctionComponent<{
     } else {
       if (region) {
         if (geoScope === GeoScopeFilter.COUNTRY)
-          return sortedLocations.slice(0, 100);
+          return modalLocations.slice(0, 100);
         else return modalLocations;
       } else {
         if (homepageScope === HomepageLocationScope.STATE)
-          return modalLocations;
+          return sortedLocations;
```

Comment:
this (prev. set as `modalLocations`) was resolving to `sortedLocations`, so i just replaced `modalLocations` with `sortedLocations`